spec/factories.rb

```diff
@@ -1,4 +1,8 @@
-FactoryGirl.define do
+FactoryGirl.define do  factory :book_author do
+    book_id 1
+author_id 1
+  end
```

Comment:
`end` at 4, 2 is not aligned with `factory :book_author do` at 1, 23 or `FactoryGirl.define do  factory :book_author do` at 1, 0.